tests/chainer_tests/functions_tests/connection_tests/test_deconvolution_2d.py

```diff
@@ -43,180 +40,90 @@ def _pair(x):
         'groups': [1, 2],
     }),
 ])))
-@backend.inject_backend_tests(
+@testing.inject_backend_tests(
```

Comment:
Backend configurations are greatly reduced. Please restore them.

Reply:
Fixed, sorry about this one.
Thanks!